build.gradle

```diff
@@ -15,10 +15,51 @@
  * limitations under the License.
  */
 
+ /* - Scalafmt Toolage -*/
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath "cz.alenkacz:gradle-scalafmt:${gradle.scalafmt.version}"
+  }
+}
+
 plugins {
     id 'com.github.blindpirate.gogradle' version '0.8.1'
 }
 
+subprojects {
+  apply plugin: 'scalafmt'
+  scalafmt.configFilePath = gradle.scalafmt.config
+
+  afterEvaluate {
```

Comment:
This should not be required. In core repo its purpose is to customize the launch classpath for controller and invoker apps. For cli builds we do not require that as there is no module with `application` plugin